pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -370,7 +370,7 @@ func withVolumeDeletionTimestamp(pvs []*v1.PersistentVolume) []*v1.PersistentVol
 }
 
 // newClaim returns a new claim with given attributes
-func newClaim(name, claimUID, capacity, boundToVolume string, phase v1.PersistentVolumeClaimPhase, class *string, annotations ...string) *v1.PersistentVolumeClaim {
+func newClaim(name, claimUID, capacity, boundToVolume string, phase v1.PersistentVolumeClaimPhase, conditions []v1.PersistentVolumeClaimCondition, class *string, annotations ...string) *v1.PersistentVolumeClaim {
```

Comment:
Please follow `withClaimVolumeMode` / `withExpectedCapacity` pattern and add `withClaimConditions`, you don't need to add a parameter to all `newClaim` calls.